src/OrleansCounterControl/app.manifest

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <asmv1:assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv1="urn:schemas-microsoft-com:asm.v1" xmlns:asmv2="urn:schemas-microsoft-com:asm.v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <assemblyIdentity version="1.2.0.0" name="OrleansCounterControl.exe"/>
+  <assemblyIdentity version="1.5.0.0" name="OrleansCounterControl.exe"/>
```

Comment:
Is this manifest of any value since it wasn't updated in a long time? /cc @jason-bragg